docs/core/building/index.md

```diff
@@ -0,0 +1,100 @@
+ * [FreeBSD](https://github.com/dotnet/coreclr/blob/master/Documentation/building/freebsd-instructions.md) 
+ * [NetBSD](https://github.com/dotnet/coreclr/blob/master/Documentation/building/netbsd-instructions.md)
+
+There is no cross-building across OS (only for ARM, which is built on X64).  
+You have to be on the particular platform to build that platform.  
+
+The build has two main `buildTypes`:
+
+ * Debug (default)- Compiles the runtime with additional runtime checks (asserts). These checks slow runtime execution but are really valuable for debugging. Recommended setting for development and testing environments.
+ * Release - Compiles the runtime without any runtime checks. It can be difficult to debug. Pass `release` to the build script to select this.
+
+In addition, by default the build not only creates the runtime executables, but it also builds all the tests.
+There are quite a few tests, taking a significant amount of time that isn't necessary if you just want to experiment with changes.
+You can skip the tests builds by adding the `skiptests` argument to the build script, like in the following example (replace `.\build` with `./build.sh` on Unix machines):
+
+```bat
+    .\build skiptests 
+```
+
+The previous example showed how to build the `Debug` flavor, which has development time checks (asserts). To build the release (full speed) flavor, do the following:
+
+```bat 
+    .\build release skiptests
+```
+
+You can find more build options with build by using the -? or -help qualifier.   
+
+### Using Your Build
+
+The build places all of its generated files under the `bin` directory at the base of the repository.
+There is a *bin\Log* directory that contains log files generated during the build (Most useful when the build fails).
+The actual output is placed in a *bin\Product\[platform].[CPU architecture].[build type]* directory, such as *bin\Product\Windows_NT.x64.Release*.
+
+While the 'raw' output of the build is sometimes useful, normally you are only interested in the NuGet packages, which are placed in the `.nuget\pkg` subdirectory of the above output directory.
+
+There are two basic techniques for using your new runtime:
+
+ 1. **Use dotnet.exe and NuGet to compose an application**.
+    See [Using Your Build](https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingYourBuild.md) for instructions on creating a program that uses your new runtime by using the NuGet packages you just created and the 'dotnet' command-line interface (CLI).  This is the expected way non-runtime developers are likely to consume your new runtime.    
+
+ 2. **Use corerun.exe to run an application using unpackaged Dlls**.
+    This repository also defines a simple host called corerun.exe that does NOT take any dependency on NuGet.
+    This host has to be told where to get the necessary DLLs you actually use, and you have to manually gather them together.
```

Comment:
Can you simplify this sentence? Maybe  "You need to tell the host where to get the required DLLs..."